.bin/bashbrew-buildkit-env-setup.sh

```diff
@@ -62,7 +62,9 @@ _bashbrew_buildkit_env_setup() {
 			vars="$(_jq_setenv <<<"$vars" BUILDX_BUILDER "$buildxBuilder")"
 
 			local sbomTag
-			sbomTag="$(grep <<<"$externalPins" -m1 '^docker/buildkit-syft-scanner:')"
+			# https://hub.docker.com/r/docker/scout-sbom-indexer/tags
+			# TODO: update to a pinned major version tag when available
```

Comment:
This comment is no longer relevant, correct?

Reply:
After talking with @yosifkit , there is a preference to follow a "stable" tag, e.g., `1` rather than `latest`.